missing pycodestyle dependency

Encountered issue while using template:

```
make test-debug
...
pycodestyle tests
make: pycodestyle: No such file or directory
make: *** [Makefile:40: test-debug] Error 127
...
```

pycodestyle used in makefile, but not specified as a dependency.

https://github.com/userver-framework/service_template/blob/a84cf8a3b0f2c56ad6f03931ac47290d8ead1a40/Makefile#L40

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -46,7 +46,8 @@ jobs:
             run: |
                 (cd third_party && git clone -b develop --single-branch --depth 1 https://github.com/userver-framework/userver.git)
                 sudo apt update
-                sudo apt install --allow-downgrades -y pycodestyle $(cat third_party/userver/scripts/docs/en/deps/${{matrix.os}}.md | tr '\n' ' ')
+                sudo apt install --allow-downgrades -y $(cat third_party/userver/scripts/docs/en/deps/${{matrix.os}}.md | tr '\n' ' ')
+                pip install -r requirements.txt
 
           - name: Setup ccache
             run: |
@@ -80,4 +81,3 @@ jobs:
             if: matrix.make == 'test-release'
             run: |
                 killall service_template
-

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,0 +1,1 @@
+pycodestyle

